### ISSUE TITLE
[DependencyInjection] Services no longer injected into __invoke controllers method

### ISSUE THREAD

**Issue by MatTheCat** [symfony/symfony#27208]
**Symfony version(s) affected**: 4.1.0-BETA1

**Description**  
After upgrading to 4.0.8 to 4.1.0-BETA1 some controllers stopped working, displaying

> Controller "Controller" requires that you provide a value for the "$argument" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one.

After some debugging it appears `ServiceValueResolver`'s container doesn't contain the controller so its `supports` method returns `false`.

**How to reproduce**  
Typehint a controller's `__invoke` method argument with a service class and then navigate to a route calling this method.

**Comment by xabbuh** [comment]
Would you mind creating a small example application that allows to reproduce?

**Comment by MatTheCat** [comment]
The service I'm trying to inject ends in `removed-ids.php` file.

**Comment by stof** [comment]
is your service tagged with `controller.service_arguments` ?

**Comment by MatTheCat** [comment]
@stof not my service but the controller. I started from the default `services.yaml` file wich contain

```yaml
SendinBlue\Controller\:
        resource: '../src/Controller'
        tags: ['controller.service_arguments']
```

**Comment by MatTheCat** [comment]
The issue really comes from `__invoke`; if I rename it services get injected correctly. But the diff between the two containers doesn't show anything probant...

**Comment by ogizanagi** [comment]
Should be fixed by #27214
